metrics/middleware.go

```diff
@@ -73,6 +75,18 @@ func (m *Middleware) CollapsePaths(flag bool) *Middleware {
 	return m
 }
 
+// HidePaths if set to true, forces the value of the "request" label
+// of the prometheus request metrics to be set to an empty value.
+// eg. (when set to true):
+//   - /decisions/service/my-service -> /hidden
+//   - /decisions -> /hidden
```

Comment:
Shouldn't this be

```suggestion
//   - /decisions/service/my-service -> /
//   - /decisions -> /
```

?

Reply:
Right, this is a leftover from my tests. At the beginning, I used a "/hidden" value for a path to validate if I see it correctly in prometheus. I will remove the example, I think the explanation mentioning "empty value" should be sufficient.